omxplayer/player.py

```diff
@@ -503,12 +509,21 @@ def quit(self):
         self._process = None
 
     @_check_player_is_active
+    def get_source(self):
+        """
+        Returns:
+            str: source currently playing
+        """
+        return self._source
+
+    # For backward compatibility
+    @_check_player_is_active
     def get_filename(self):
         """
         Returns:
-            str: filename currently playing
+            str: source currently playing, for backward compatibility
```

Comment:
I tried a couple options, and built the docs entering `./docs` and running `make dirhtml` to check that it looked OK and that the link worked. But I'm not that proficient with Sphinx docs, please check that it's looks how supposed to.